evalml/tests/automl_tests/test_automl.py

```diff
@@ -216,6 +223,61 @@ def test_search_results(X_y_regression, X_y_binary, X_y_multi, automl_type, obje
     )
 
 
+def test_search_batch_times(caplog):
+    caplog.clear()
+    X, y = load_data(
```

Comment:
We can use a smaller dataset here: check out the `X_y_binary` fixture in `contest.py` and just search for tests that use it as an example.